File: app/src/main/java/com/ma/lightweather/app/Contants.java
package com.ma.lightweather.app;

/**
 * Created by Ma-PC on 2016/12/12.
 */
public class Contants {

    //中国和全球付费
    //public final static String BASE_URL="https://api.heweather.com/v5/";
    //中国免费
    public final static String BASE_URL="https://free-api.heweather.com/s6/";
    //全部数据
    public final static String WEATHER_ALL=BASE_URL+"weather?key=4ddad270cdb94639ae5f4cfe11aadf82&location=";
    //必应图片
    public final static String BINGURL="http://guolin.tech/api/bing_pic";
    //百度逆地理
    public final static String BAIDUGETADDRESS="http://api.map.baidu.com/geocoder/v2/?" +
            "output=json" +
            "&pois=1" +
            "&latest_admin=1" +
            "&ak=fGsdz2W5duRKjf1Gafg2nQRhgkbm7Tad" +
            "&mcode=FD:DB:F3:34:1C:AF:37:38:10:8A:38:44:06:BC:52:10:D7:41:CD:69;com.ma.lightweather"+
            "&location=";

    public final static String CITY="city";
    public final static String TMP="tmp";
    public final static String TXT="txt";
    public final static String MODEL="model";
    public final static String LOCTION="loction";
    public final static String WEATHER="weather";
    public final static String CITYNAME="洛阳";
    public final static String THEME="themetag";
    public final static String NOTIFY="notify";
    public final static String STATUS="status";
    public final static String BINGIMG="bingimg";
    public final static String LIFE="life";
    public final static String FIRSTSHOW="firstshow";
}

File: app/src/main/java/com/ma/lightweather/app/Contants.kt
package com.ma.lightweather.app

/**
 * Created by Ma-PC on 2016/12/12.
 */
object Contants {

    //常规天气
    const val WEATHER_ALL ="https://free-api.heweather.com/s6/weather?key=4ddad270cdb94639ae5f4cfe11aadf82&location="
    //空气质量
    const val WEATHER_AIR = "https://free-api.heweather.net/s6/air/now?key=4ddad270cdb94639ae5f4cfe11aadf82&location="
    //行政区域
    const val AREAURL = "https://restapi.amap.com/v3/config/district?key=233dae1280c8b4c934e1a3928fdb1f10&subdistrict=3&extensions=base&keywords="
    //必应图片
    const val BINGURL = "http://guolin.tech/api/bing_pic"

    const val WEATHER_JSON = "weather_json"
    const val WEATHER_AQI_JSON = "weather_aqi_json"
    const val TMP = "tmp"
    const val TXT = "txt"
    const val DIR = "dir"
    const val SC = "sc"
    const val DATE = "date"
    const val QLTY = "qlty"
    const val MODEL = "model"
    const val LOCTION = "loction"
    const val WEATHER = "weather"
    const val WEATHERDB = "Weather.db"
    const val CITY = "cityName"
    const val CITYCODE = "cityCode"
    const val THEME = "themetag"
    const val NOTIFY = "notify"
    const val STATUS = "status"
    const val LIFE = "life"
    const val OLDVERSION = "version"
    const val BING = "bing"
    const val BINGPATH = "bingpath"

    //和风 KEY
    const val HF_WEATHER_KEY ="dbccdd9686e54c739cea4735879b7ae8"
    //和风 城市天气
    const val HF_WEATHER_AREA ="https://geoapi.qweather.com/v2/city/lookup?key=$HF_WEATHER_KEY&location="
    //和风 实时天气
    const val HF_WEATHER_NOW ="https://devapi.qweather.com/v7/weather/now?key=$HF_WEATHER_KEY&location="
    //和风 未来天气
    const val HF_WEATHER_FUTURE = "https://devapi.qweather.com/v7/weather/7d?key=$HF_WEATHER_KEY&location="
    //和风  24小时
    const val HF_WEATHER_HOUR = "https://devapi.qweather.com/v7/weather/24h?key=$HF_WEATHER_KEY&location="
    //彩云 KEY
    const val CY_WEATHER_KEY ="7iuyQv4ve1RAEBpS"
    //彩云 全部天气
    const val CY_WEATHER_AREA ="https://api.caiyunapp.com/v2.6/$CY_WEATHER_KEY/116.3176,39.9760/weather?alert=true&dailysteps=1&hourlysteps=24"
}
